runtime/vm/jvminit.c

```diff
@@ -3989,6 +3989,10 @@ registerVMCmdLineMappings(J9JavaVM* vm)
 	if (registerCmdLineMapping(vm, MAPOPT_XXINITIALHEAPSIZE_EQUALS, VMOPT_XMS, EXACT_MAP_WITH_OPTIONS) == RC_FAILED) {
 		return RC_FAILED;
 	}
+	/* Map -XX:OnOutOfMemoryError= to -Xdump:tool:events=systhrow,filter=java/lang/OutOfMemoryError,exec= */ 
+	if (registerCmdLineMapping(vm, MAPOPT_XXONOUTOFMEMORYERROR_EQUALS, VMOPT_XDUMP_TOOL_OUTOFMEMORYERROR_EXEC_EQUALS, EXACT_MAP_WITH_OPTIONS) == RC_FAILED) {
+		return RC_FAILED;
```

Comment:
Do you have a test case for this feature?

Reply:
I will add a test to check for failure cases of this feature. The JTReg test `Basic.java` that is currently failing in the associated issue (https://github.com/eclipse/openj9/issues/4945) will pass with this feature implemented - would this be sufficient for testing the success case of this feature?